data/beps/beps.yml

```diff
@@ -634,15 +635,19 @@
         family-names: Moia
     -   given-names: Sourav
         family-names: Kulkarni
+    bids_maintainers:
+    -   given-names: Seyed Yahya
+        family-names: Shirazi
```

Comment:
@neuromechanist I see you commented on the [google doc](https://docs.google.com/document/d/1oTfjzY5ZnLIYd0kPPWhR81sBmMuy_jC5YYIaqj6OhSA/edit?pli=1&tab=t.0). Can I take that as an indication of your dedication to the peripheral physio BEP? ;)